doc/en/components/charts/types/bubble-chart.md

```diff
@@ -20,8 +22,7 @@ $PlatformShort$ Bubble Chart is a group of series that show markers with variabl
 
 ## $PlatformShort$ Bubble Chart with Single Series
 
-The Bubble Chart contains many of the characteristics of the Scatter Point Chart but with the option to have various radius scale sizes.
-
+The Bubble Chart contains many of the characteristics of the [Scatter Marker Chart](scatter-chart.md) but with the option to have various radius scale sizes.
```

Comment:
append "#$PlatformShort$-scatter-marker-chart" after ".md" to make this a link to a section of that topic

Reply:
pending